kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/system/SystemService.java

```diff
@@ -298,7 +298,7 @@
 
     /**
      * Returns the maximum number of snapshots to be retained in the file system.
-     * When the maximum number is reached, a garbage collector will deleted the older snapshots.
+     * When the maximum number is reached, a garbage collector will delete the older snapshots.
```

Comment:
Did that already, it's just not shown as a change here, because the comment is on another line.